src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -342,4 +352,25 @@ public boolean containsQualifier(@NonNull String jsonFileName) {
     public void removeQualifier(@NonNull String jsonFileName) {
         qualifiers.remove(jsonFileName);
     }
+
+    /**
+     * Exclude the specified {@link ChartType} from being rendered.
+     *
+     * @param chartTypes charts not to be rendered
+     */
+    public void chartsNotToRender(@NonNull ChartType... chartTypes) {
+        for (ChartType chartType : chartTypes) {
```

Comment:
and then https://docs.oracle.com/javase/7/docs/api/java/util/List.html#removeAll(java.util.Collection)

Reply:
If I assume you prefer `Set` versus `varargs`, then change this accordingly. to remove passed in Set...

Reply:
Renamed to what I believe is a more appropriate name.